saleor/settings.py

```diff
@@ -72,6 +72,10 @@ def get_bool_from_env(name, default_value):
     )
 }
 
+DATABASES["default"]["HOST"] = (
+    DATABASES["default"]["HOST"].replace("%3a", ":").replace("%3A", ":")
```

Comment:
Does it have to be done by `replace` here or could we use `urllib.parse.unquote` instead? Are there any escape sequences we don't want to decode?

Reply:
It's a good question. I initally thought that made sense too, but it seems from the conversation here that folks were worried about escaping too much: https://github.com/jacobian/dj-database-url/pull/67

Reply:
That said, I'm happy to try it if you think it's better. Let me know.

Reply:
Interestingly, the linked discussion points to a spec that explicitly disallows the colon from being used as part of the host. Now that I think about this, the socket path should not be a part of the hostname at all, it should look like the `file` schema:

```
cloudsql:///cloudsql/<project_id>:<region>:<instance_id>
```

(notice the three slashes, the hostname is empty and the path is used instead)